docs/rules/id-blacklist.md

```diff
@@ -77,6 +77,24 @@ foo.callback(); // all function calls are ignored
 foo.data; // all property names that are not assignments are ignored
 ```
 
+You can also provide regular expressions in place of static identifier names. Your regular expressions will automatically be assumed to span from the start to the end of the identifier name, so you do not need to explicitly use the `^` or `$` control characters in your expressions.
+
+Examples of **incorrect** code for this rule with sample `"ref\\d*"` restricted identifiers:
```

Comment:
Not a big deal, but I don't think the "`with sample "ref\\d*" restricted identifiers`" part of the sentence is needed.
We usually let the example code snippets speak for themselves, since they have the `/*eslint id-blacklist ...*/` comment in them.